run.sh

```diff
@@ -163,6 +163,8 @@ fi
 
 if [[ "${OS}" == "Linux" ]]
 then
+  sudo apt update && sudo apt upgrade -y
+
```

Comment:
Moet hier een waarschuwing boven? Duurt dit lang? Kan de output `> /dev/null`?

Reply:
Waarschuwing had ik inderdaad al gedaan, output gaat nu richting `/dev/null` Willen we dat ook voor de andere installs?

Reply:
Niet per se, het gaat vooral omdat dit een automatische is bij elke run dus altijd troep oplevert. Bij install van missende dingen lijkt het me wel goed om output te geven.